src/webapp/components/survey/hook/useSaveSurvey.ts

```diff
@@ -16,7 +16,7 @@ export function useSaveSurvey(formType: SURVEY_FORM_TYPES, orgUnitId: Id, survey
         switch (formType) {
             case "PPSWardRegister":
             case "PPSPatientRegister":
-                return (orgUnitId = currentHospitalForm?.orgUnitId ?? "");
+                return currentHospitalForm?.orgUnitId ?? "";
```

Comment:
As before, we have to be careful with theses `?? ""`, which are handy but can obscure errors. The handing depends on the particular situation, as before.

Reply:
the "" is handled in https://github.com/EyeSeeTea/amr-surveys/blob/2a4998b132ae1f4aade610a93a2ac92f509d6a4b/src/domain/usecases/SaveFormDataUseCase.ts#L22. Does that make sense?

Reply:
Typically, when using an empty string to denote "missing value", the type what we really want is `string | undefined.